tests/storages_tests/test_storages.py

```diff
@@ -1204,6 +1206,59 @@ def test_retry_failed_trial_callback(storage_mode: str, max_retry: Optional[int]
         assert RetryFailedTrialCallback.retried_trial_number(study.trials[1]) == (
             None if max_retry == 0 else 0
         )
+        # Test inheritance of trial fields
```

Comment:
A nit, but we usually use full sentences, so could you put a period at the end of the line?
c.f., https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#block-and-inline-comments-are-supposed-to-be-sentence-s-
```suggestion
        # Test inheritance of trial fields.
```